event-stream-service/charts/event-stream-service/values.yaml

```diff
@@ -0,0 +1,204 @@
+ess:
+  nats:
+    secret:
+      name: ess-nats-auth
+  nginx:
+    route:
+      name: ess-nginx-route
+      enabled: true
+      ingressPrefix: ess-
+      ingressSuffix: a191b5-dev.apps.silver.devops.gov.bc.ca
+      path: ""
+      serviceName: ess-nginx
+      targetPort: http
+      timeout: 2m
+      tls:
+        enabled: true
+        insecureEdgeTerminationPolicy: None
+        termination: edge
+      wildcardPolicy: None
+
+nats:
+  fullnameOverride: ess-nats
+  config:
+    cluster:
+      enabled: true
+      replicas: 2
```

Comment:
Should this be three replicas, to be consistent with how OpenShift HA is usually done?